examples/onnxruntime/training/text-classification/zero_stage_2.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Could you also elaborate in the readme or in the script (eg. with `deepspeed` argument of `ORTTrainer`) how this config file would be used?